### ISSUE TITLE
create apps from zero

### ISSUE THREAD

**Issue by willzhang** [fluxcd/flux2#541]
the Get Started docs Mix  install fux and delploy app together，it Confused me，why don't you separate them，i can't distinguish what steps is for install flux ,and what steps is for deploy apps.

how to create an hello world apps from zero?

must i use flux comand generate yamlfile?

must i create source git use flux ?

i have a git repo and a path with some app yamlfiles, and i have installed flux , then ,how can i delploy them from zero?

why install fluxcd use a github or gitlab repo,  i think it will be a good way install  fluxcd standalone, like

```
kubectl create ns fluxcd-system
kubectl apply -f fluxcd.yaml
```

then you can use some Different chapters tell people how to integrate with git repo,

* step1: install fluxcd

* step2: integrate  with git repo (optional)

* step3: deploy you personal apps

**Comment by stefanprodan** [comment]
Yes we are considering rewriting the "get started" guide and point users to [flux2-kustomize-helm-example](https://github.com/fluxcd/flux2-kustomize-helm-example) and [flux2-multi-tenancy](https://github.com/fluxcd/flux2-multi-tenancy) for more advanced examples.

**Comment by Marx2** [comment]
I totally agree. I have a repo with yamls and just wanted to make flux manage them. Unfortunately bootstrap fails on existing repo (cannot find repository for push, while it cloned it a second ago). --verbose flag doesn't change output at all, so I'm not able to debug it.
Another strange behaviour is when I've created new repo with bootstrap command, it committed 3 empty yamls. I don't understand why did it make them empty?
Anyway, I wanted to find where flux stores git address, but failed. I've browsed deployments, pods, secrets, configmaps, but none of them contains this URL.
I really would prefer to generate yaml and apply them to cluster, instead of cryptic command which makes something somewhere and I have no control or knowledge what happens.
